### DISCUSSION TITLE
Nucleo32 L432 as game joystick

### DISCUSSION THREAD

**Discussion by Kamoool** [stm32duino/Arduino_Core_STM32#1524] [Libraries] [answered]
Hi,
I would like to use nucleo32 L432 as PC game joystick which reads data from several potentiometers and buttons. Is there already HID joystick library for STM32duino?

**Comment by fpistm** [comment, marked as answer]
Hi @Kamoool 
currently not. Only HID Mouse and Keyboard.
It should be possible to add Joystick descriptor.
